src/App.tsx

```diff
@@ -16,35 +22,96 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+
+  const handleReverse = () => {
+    setIsReversed(!isReversed);
+  };
```

Comment:
[CODE STYLE] - The function name `handleReverse` doesn't clearly describe what it does. A more descriptive name might be `toggleReverse`.